docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md

```diff
@@ -1,11 +1,79 @@
 ---
-title: How to configure credentials using a secrets store 
+title: How to configure credentials
 ---
 import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
 import Tabs from '@theme/Tabs'
 import TabItem from '@theme/TabItem'
 
-Choose which Secret Manager you are using:
+This guide will explain how to configure your ``great_expectations.yml`` project config to populate credentials from either a YAML file or a secret manager.
+
+<Tabs
+  groupId="yaml-or-secret-manager"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Secret Manager', value:'secret-manager'},
+  ]}>
+
+<TabItem value="yaml">
+
+<Prerequisites>
+
+  - [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
```

Comment:
```suggestion
```
You can just leave the defaults from `Prerequisites` - otherwise it's redundant (the intro link is already here):
```
Prerequisites: This how-to guide assumes you have:

    Completed the Getting Started Tutorial
    Have a working installation of Great Expectations
    Set up a working deployment of Great Expectations
```

Reply:
✅